src/main/webapp/go.js

```diff
@@ -105,7 +108,11 @@ function addScriptToHead() {
  * to reflect the current state of the hunt.
  */
 function getHunt() {
-  fetch(DATA_URL).then((response) => response.json()).then((mssg) => {
+  // ID of the scavenger hunt that the user is on,
+  // in the form of: hunt_id=ScavengerHunt([ID number here])
+  huntID = window.location.search.substr(1);
```

Comment:
You should note that you're using substr(1) here because the first character in window.location.search is '?'